pkg/types/args_test.go

```diff
@@ -0,0 +1,92 @@
+package types_test
+
+import (
+	"reflect"
+
+	"github.com/appc/cni/pkg/types"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("UnmarshallableBool UnmarshalText", func() {
```

Comment:
Mostly a matter of style but we should be consistent with our test files across the board.